ibmcloudshellv1/ibm_cloud_shell_v1_integration_test.go

```diff
@@ -0,0 +1,173 @@
+
+			Expect(err).To(BeNil())
+			Expect(ibmCloudShellService).ToNot(BeNil())
+			Expect(ibmCloudShellService.Service.Options.URL).To(Equal(serviceURL))
+		})
+	})
+
+	Describe(`GetAccountSettingsByID - Get account settings`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`GetAccountSettingsByID(getAccountSettingsByIdOptions *GetAccountSettingsByIdOptions)`, func() {
+
+			getAccountSettingsByIdOptions := &ibmcloudshellv1.GetAccountSettingsByIdOptions{
+				AccountID: core.StringPtr("12345678-abcd-1a2b-a1b2-1234567890ab"),
+			}
+
+			accountSettings, response, err := ibmCloudShellService.GetAccountSettingsByID(getAccountSettingsByIdOptions)
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(accountSettings).ToNot(BeNil())
+
+		})
+	})
+
+	Describe(`UpdateAccountSettingsByID - Update account settings`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`UpdateAccountSettingsByID(updateAccountSettingsByIdOptions *UpdateAccountSettingsByIdOptions)`, func() {
+
+			featureModel := []ibmcloudshellv1.Feature{
+				{
+					Enabled: core.BoolPtr(true),
+					Key:     core.StringPtr("server.file_manager"),
+				},
+				{
+					Enabled: core.BoolPtr(true),
+					Key:     core.StringPtr("server.web_preview"),
+				},
+			}
+
+			regionSettingModel := []ibmcloudshellv1.RegionSetting{
+				{
+					Enabled: core.BoolPtr(true),
+					Key:     core.StringPtr("eu-de"),
+				},
+				{
+					Enabled: core.BoolPtr(true),
+					Key:     core.StringPtr("jp-tok"),
+				},
+				{
+					Enabled: core.BoolPtr(true),
+					Key:     core.StringPtr("us-south"),
+				},
+			}
+
+			accountID := "12345678-abcd-1a2b-a1b2-1234567890ab"
+			updateAccountSettingsByIdOptions := &ibmcloudshellv1.UpdateAccountSettingsByIdOptions{
+				AccountID:                   core.StringPtr(accountID),
+				NewID:                       core.StringPtr("ac" + accountID),
+				NewRev:                      core.StringPtr("130" + accountID),
+				NewAccountID:                core.StringPtr(accountID),
+				NewCreatedAt:                core.Int64Ptr(int64(1600079615)),
+				NewCreatedBy:                core.StringPtr("IBMid-1000000000"),
+				NewDefaultEnableNewFeatures: core.BoolPtr(true),
+				NewDefaultEnableNewRegions:  core.BoolPtr(true),
+				NewEnabled:                  core.BoolPtr(true),
+				NewFeatures:                 featureModel,
+				NewRegions:                  regionSettingModel,
+				NewType:                     core.StringPtr("account_settings"),
+				NewUpdatedAt:                core.Int64Ptr(int64(1624359948)),
+				NewUpdatedBy:                core.StringPtr("IBMid-1000000000"),
+			}
+
+			accountSettings, response, err := ibmCloudShellService.UpdateAccountSettingsByID(updateAccountSettingsByIdOptions)
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(accountSettings).ToNot(BeNil())
+
```

Comment:
Probably wouldn't hurt to add a few more Expect() statements here to verify that certain fields were in fact updated (e.g. Features, Regions, etc.).

Reply:
Done